index.html

```diff
@@ -6,15 +6,164 @@
+            </div>
+        </div>
+
+        <div class="row justify-content-center main-row2 ">
+            <div class="col-md-4 aboutCol">
+                <h3>About</h3>
+                <h2>Maria Hope</h2>
+                <p class="para1">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quidem deserunt quasi fuga explicabo laboriosam quo nihil doloremque dolores architecto corrupti fugiat pariatur nesciunt saepe consectetur veritatis, tenetur, possimus esse tempore?</p>
+                <div class="col-12 d-flex align-items-center arrow">
+                    <span>&#129058;</span>
+                </div>
+                <div class="col d-flex justify-content-center">
+                    <img class="col Maria-img"
+                        src="https://static.wixstatic.com/media/9c608a_2a78cf5937414d0e9a597504d68daafc~mv2_d_6046_4035_s_4_2.jpg/v1/crop/x_776,y_0,w_3647,h_4035/fill/w_519,h_572,al_c,q_80,usm_0.66_1.00_0.01/9c608a_2a78cf5937414d0e9a597504d68daafc~mv2_d_6046_4035_s_4_2.webp"
+                        alt="image of Maria">
+                    </div>
+                </div>
+                
+ 
+        </div>
 
-    <!-- Add your markup here -->
+        <div class="row main-row3">
+            <div class="col-md-4 item-1">
+                <h4>01.</h4>
+                <p><b>Individual Therapy</b></p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos et sunt reprehenderit labore, odit aliquid maiores ipsa minus consequuntur deleniti iure distinctio impedit a consectetur sequi ut error doloremque omnis?</p>
+            </div>
+            <div class="col-md-4 item-2">
+                <h4>02.</h4>
+                <p ><b>Group Therapy</b></p>
+                <p>Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ex harum itaque doloribus fuga dicta eligendi! Voluptas quaerat debitis officiis repellendus beatae, nostrum rem maiores ipsum a fugiat iste vel commodi?</p>
+                <div class="wrapper">
+                    <button type="button" class="btn-2">View more services</button>
+                    <span id="arrow2">&#129058;</span>
+                </div>
+                
+            </div>
+            <div class="col-md-4 item-3">
+                <h4>03.</h4>
+                <p><b>Couples Therapy</b></p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestiae, voluptatibus quae, voluptatum veniam eveniet consequatur voluptate rerum dolorem suscipit ullam repudiandae. Labore, perspiciatis rerum minima praesentium recusandae molestiae eaque pariatur?</p>
+            </div>
+        </div>
+
+        <div class="row justify-content-center main-row4 item-4">
+            <h3 id="articles-text">Articles about Psychology</h3>
+                <div class="col-md-2 ">
+                    <div class="card" style="width: 18rem;">
```

Comment:
Don't set styling inline like this - again an issue with specificity. Keep your styles in your CSS file.

Also in this case, you'd be better off setting your `card` class to `width: 100%` and letting its `col-...` parent (on line 112) determine how wide it gets: for mobile, either `col-12` or `col-10 offset-1` for example, and then for tablet you might want `col-md-4` as there are 3 of them and you want your columns to always add up to 12. 

As mentioned above, if you remove the fixed heights on this blog section and the services section above it, elements should flow a little better on mobile.